test/api/site/explore_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'api_test_helper'
+
+class SearchControllerTest < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+  include Requests::JsonHelpers
+  include Requests::HttpHelpers
+
+  HOMO_SAPIENS_FILTER = { id: 'NCBITaxon_9606', name: 'Homo sapiens' }
+  NO_DISEASE_FILTER = { id: 'MONDO_0000001', name: 'disease or disorder' }
```

Comment:
Can we talk through the intent of NO_DISEASE_FILTER? I suspect its inclusion in this test is related to its function in search_controller_test.rb and I want to make sure that what we're doing is still appropriate.

Reply:
Ooops, this was a copy-paste error -- this doesn't belong here.  I just copied the test structure of this test from search_controller (as you guessed).  I'll delete